README.md

```diff
@@ -15,6 +15,35 @@ __Jump To:__
 
 ## Installation
 
+### Supported Architectures
+
+#### Linux
+- manylinux2014-x64
+- manylinux2014-x86
+- al2-x64
+- alpine-3.16-x64
+- alpine-3.16-x86
+- alpine-3.16-armv6
+- alpine-3.16-armv7
+- alpine-3.16-arm64
+- raspberry pi
```

Comment:
We probably would like to mention raspberry pi **bullseye** as that is the only raspberry pi os we tested against.